lib/moon/design/icon_button.ex

```diff
@@ -0,0 +1,65 @@
+defmodule Moon.Design.IconButton do
```

Comment:
shouldn't we name the module Moon.Design.Button.Icon and put it to appropriate folder?